tests/v2/actions/workloads/deployment/verify.go

```diff
@@ -34,3 +37,52 @@ func VerifyDeployment(steveClient *steveV1.Client, deployment *steveV1.SteveAPIO
 
 	return err
 }
+
+// VerifyRolloutHistoryDeployment is a helper to validate rollout history deployment
+func VerifyRolloutHistoryDeployment(client *rancher.Client, clusterID, namespaceName string, deploymentName string, expectedRevision string) error {
```

Comment:
our verifies usually now do the wait() on the object, but in this case i think the object is already up and we're validating that its matching an expected 'object' (in this case a string)


how would we feel about giving these a structured name that's slightly different than our normal validates so that we can tell at a glance what it should do
something like `VerifyDeploymentAgainstRolloutHistory` or `VerifyRolloutHistoryAgainstExistingRevision`
-> something that would signify "we're expecting an object to validate against"